docs/ssh.md

```diff
@@ -106,7 +106,7 @@ that this is a Pod-wide setting, unlike the container `securityContext` above.
 If you want git-sync to run as a different (non-root) UID and GID, you can
 change these last blocks to any UID/GID you like.  SSH demands that the current
 UID be present in /etc/passwd, so in this case you will need to add the
-`--add-user` flag to git-sync's args array.
+`--add-user` flag to git-sync's args array. Also, you need to change `GIT_SYNC_ROOT` to some other location, say `/workspace`, instead of `$HOME/git` as the `$HOME` will be `/` for the user and, `GIT_SYNC_ROOT` will end up in `//tmp` which do not exist.
```

Comment:
Super nit-pick - can you please linewrap to match the rest? :)

Reply:
Also "`//tmp` which do not exist" -> "`//git` which the non-root user can't create"